microcosm_postgres/dag.py

```diff
@@ -121,6 +144,24 @@ def clone(self, substitutions, **kwargs):
         cloned_dag = dag.clone(ignore=self.ignore)
         return self.update_nodes(self.add_edges(cloned_dag))
 
+    def replace_dag(self, nodes_map, edges, **kwargs):
+        """
+        Create objects from an input DAG.
+        Assuming all relevant stores are accessible.
+
+        """
+        nodes = [
+            getattr(self, '{}_store'.format(model_name)).model_class(**node)
```

Comment:
This is a bit hacky because of the resource deserialization. Happy to change if there are better generic alternatives

Reply:
I'd do a few things here:
 - Use a format string!
 - Make the store lookup for a model its own internal function so it can be changed safely. Odds are this will fail without an `underscore(model_name)`...
 - See if there's an easy way to have the dag nodes know their type